Testing/SystemTests/tests/analysis/reference/EQSANSTOFStructureTest.py

```diff
@@ -0,0 +1,42 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
I just realized after merging :man_facepalming: this should be in` Testing/SystemTests/tests/analysis` instead of `Testing/SystemTests/tests/analysis/reference`.